airflow-core/src/airflow/ui/src/theme.ts

```diff
@@ -34,140 +34,336 @@ const customConfig = defineConfig({
+          "800": { value: "oklch(27.8% .033 256.848)"},
+          "900": { value: "oklch(21% .034 264.665)"},
+          "950": { value: "oklch(13% .028 261.692)"},
+        },
+        zinc: {
+          "50":  { value: "oklch(98.5% 0 0)"},
+          "100": { value: "oklch(96.7% .001 286.375)"},
+          "200": { value: "oklch(92% .004 286.32)"},
+          "300": { value: "oklch(87.1% .006 286.286)"},
+          "400": { value: "oklch(70.5% .015 286.067)"},
+          "500": { value: "oklch(55.2% .016 285.938)"},
+          "600": { value: "oklch(44.2% .017 285.786)"},
+          "700": { value: "oklch(37% .013 285.805)"},
+          "800": { value: "oklch(27.4% .006 286.033)"},
+          "900": { value: "oklch(21% .006 285.885)"},
+          "950": { value: "oklch(14.1% .005 285.823)"},
+        },
+        neutral: {
+          "50":  { value: "oklch(98.5% 0 0)"},
+          "100": { value: "oklch(97% 0 0)"},
+          "200": { value: "oklch(92.2% 0 0)"},
+          "300": { value: "oklch(87% 0 0)"},
+          "400": { value: "oklch(70.8% 0 0)"},
+          "500": { value: "oklch(55.6% 0 0)"},
+          "600": { value: "oklch(43.9% 0 0)"},
+          "700": { value: "oklch(37.1% 0 0)"},
+          "800": { value: "oklch(26.9% 0 0)"},
+          "900": { value: "oklch(20.5% 0 0)"},
+          "950": { value: "oklch(14.5% 0 0)"},
+        },
+        stone: {
+          "50":  { value: "oklch(98.5% .001 106.423)"},
+          "100": { value: "oklch(97% .001 106.424)"},
+          "200": { value: "oklch(92.3% .003 48.717)"},
+          "300": { value: "oklch(86.9% .005 56.366)"},
+          "400": { value: "oklch(70.9% .01 56.259)"},
+          "500": { value: "oklch(55.3% .013 58.071)"},
+          "600": { value: "oklch(44.4% .011 73.639)"},
+          "700": { value: "oklch(37.4% .01 67.558)"},
+          "800": { value: "oklch(26.8% .007 34.298)"},
+          "900": { value: "oklch(21.6% .006 56.043)"},
+          "950": { value: "oklch(14.7% .004 49.25)"},
         },
       },
     },
     semanticTokens: {
       colors: {
-        success: generateSemanticTokens("success"),
-        failed: defaultConfig.theme?.semanticTokens?.colors?.red ?? {},
-        queued: generateSemanticTokens("queued"),
-        skipped: defaultConfig.theme?.semanticTokens?.colors?.pink ?? {},
-        up_for_reschedule: defaultConfig.theme?.semanticTokens?.colors?.cyan ?? {},
-        up_for_retry: defaultConfig.theme?.semanticTokens?.colors?.yellow ?? {},
-        upstream_failed: defaultConfig.theme?.semanticTokens?.colors?.orange ?? {},
-        running: generateSemanticTokens("running"),
-        restarting: generateSemanticTokens("restarting"),
-        deferred: generateSemanticTokens("deferred"),
-        scheduled: generateSemanticTokens("scheduled"),
-        none: generateSemanticTokens("none", "black"),
-        removed: generateSemanticTokens("removed", "black"),
+        navbar: generateSemanticTokens("gray"),
```

Comment:
Personally, I am not a fan of a gray nav bar

Reply:
Yeah, we got rid of that config in 3.0.0, because we need to make them dark mode compatible. Maybe the config value is a dict for light and dark?

Reply:
We could leverage a JS script like [material-color-utilities](https://github.com/material-foundation/material-color-utilities) to generate all shades from an input color.

Reply:
@bbovenzi : check this out https://jsfiddle.net/xs9ut61h/#&togetherjs=KHogmc9BJM
The missing piece of the puzzle is reading from AIRFLOW__WEBSERVER__NAVBAR_COLOR and calling generatePalette